dev-bin/make-man-pages.pl

```diff
@@ -2,19 +2,18 @@
 
 use strict;
 use warnings;
-use autodie qw( :all );
+use autodie;
```

Comment:
If we remove the `:all`, it would probably be worth including `$? >> 8` in the `system` error message as the exit code may be useful in diagnosing a failure.

Reply:
I'm happy to do that, but also happy to keep the `:all` as well (it has a dependency `IPC::System::Simple`, but that's not a big problem). I was just trying to converge with geoipupdate a bit, which doesn't have `:all` and has those `or die` in system. So, no strong feelings myself, and happy to go with whatever you think is best!

Reply:
Yeah, I am not sure why these two scripts diverged on `autodie`. Using `:all` seems the simplest to me. The `geoipupdate` script appears to check every `binmode`, `print`, `close`, etc., which seems excessive when we can just use `autodie`.